software/webapp/src/main/java/org/apache/brooklyn/entity/dns/AbstractGeoDnsServiceImpl.java

```diff
@@ -180,6 +180,7 @@ protected void endTracker() {
     public static class MemberTrackingPolicy extends AbstractMembershipTrackingPolicy {
         @Override
         protected void onEntityEvent(EventType type, Entity entity) {
+            defaultHighlightAction(type, entity);
```

Comment:
Not needed as this policy extends `AbstractMembershipTrackingPolicy ` and `onEntityEvent` [already call `defaultHighlightAction`](https://github.com/apache/brooklyn-server/blob/master/core/src/main/java/org/apache/brooklyn/entity/group/AbstractMembershipTrackingPolicy.java#L242)